frontend/express/public/stylesheets/main.css

```diff
@@ -2841,4 +2832,380 @@ color:#E95C6C; transition:color 1s;
+    font-size: 12px;
+    margin-bottom: 8px;
+    color: #636363;
+}
+
+.graph-note-create .colors .color {
+    border-radius: 2px;
+    float: left;
+    width: 20px;
+    height: 20px;
+    position: relative;
+    margin-right: 10px;
+    cursor: pointer;
+}
+.graph-note-create .colors .color.alt1 {
+    background-color: #79a3e9;
+}
+.graph-note-create .colors .color.alt2 {
+    background-color: #70bbb8;
+}
+.graph-note-create .colors .color.alt3 {
+    background-color: #e2bc33;
+}
+.graph-note-create .colors .color.alt4 {
+    background-color: #a786cd;
+}
+.graph-note-create .colors .color.alt5 {
+    background-color: #dd6b67;
+}
+.graph-note-create .colors .color.alt6 {
+    background-color: #ece176;
+}
+
+.graph-note-create .colors .color.selected:after {
+    font-family: Ionicons;
+    content: "\f3fd";
+    position: absolute;
+    top: -2px;
+    font-size: 23px;
+    color: #fff;
+    left: 5px;
+}
+
+.graph-note-create .fields .note::placeholder{
+    font-size:14px;
+}
+.graph-note-create .fields .note {
+    border-radius: 2px;
+    border-color: #dcdbdb;
+    display: block;
+    width: calc(100% - 14px);
+    z-index: auto;
+    position: relative;
+    line-height: normal;
+    font-size: 11px;
+    transition: none 0s ease 0s;
+    background: #fff !important;
+    resize: none;
+    font-size:14px;
+    padding-left: 10px;
+}
+
+.graph-note-create .button {
+    visibility: visible;
+    cursor: pointer;
+    user-select: none;
+    background-color: #ececec;
+    float: left;
+    border: 1px solid #d0d0d0;
+    margin-right: -1px;
+    color: #636363;
+    width: 68px;
+    height: 30px;
+    text-align: center;
+    width: 32.8%;
+    line-height: 30px;
+    font-size: 13px;
+}
+.graph-note-create .note-types-selector .button:nth-child(2) {
+    border-top-left-radius: 2px;
+    border-bottom-left-radius: 2px;
+}
+.graph-note-create .button:last-child {
+    border-top-right-radius: 2px;
+    border-bottom-right-radius: 2px;
+    border-right: 1px solid #d0d0d0;
+}
+.graph-note-create .calendar {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+.graph-note-create .date-time-selector-container i {
+    font-size: 14px;
+    margin: 5px 5px;
+}
+.graph-note-create .date-time-selector-container .date-time-value-show {
+    flex-grow: 1;
+    padding-left: 10px;
+}
+.graph-note-create .button.active {
+    background-color: #ffffff;
+}
+.graph-note-create .selectize-control.multi .selectize-input > div {
+    padding: 3px 22px 3px 10px !important;
+    line-height: 14px !important;
+    background-color: #777;
+    color: #f1f1f1;
+}
+.note-table-view-navigator .back-link {
+    cursor: pointer;
+    margin-bottom: 25px;
+    color: #616161;
+    font-size: 12px;
+    display: inline-block;
+    text-transform: uppercase;
+}
+.note-table-view-navigator .back-link:before {
+    content: "←";
+    margin-right: 5px;
+    color: #616161;
+}
+
+.date-time-selector-container {
```

Comment:
This seems to be too broad class, can you prefix it with some container, so we know it only applies where you expect it to apply?